pkgs/applications/audio/musescore/default.nix

```diff
@@ -64,6 +64,14 @@ in stdenv'.mkDerivation rec {
       url = "https://github.com/doronbehar/MuseScore/commit/f48448a3ede46f5a7ef470940072fbfb6742487c.patch";
       hash = "sha256-UEc7auscnW0KMfWkLKQtm+UstuTNsuFeoNJYIidIlwM=";
     })
+    # Upstream removed the option to use system freetype in v4.1.0,
+    # which caused crashes as the bundled version is outdated, see #244409.
+    # For now apply the patch manually. The fix has been merged upstream,
+    # so we can remove this patch with the next version.
```

Comment:
Links that I think are important to put here:

- https://github.com/musescore/MuseScore/issues/11572
- https://github.com/musescore/MuseScore/issues/18795